api/RTCPeerConnection.json

```diff
@@ -1736,13 +1444,16 @@
             "description": "Uses callbacks instead of promises",
```

Comment:
Can you delete this subfeature and use the same `partial_implementation` ranges you'll be used for other methods that went from callbacks-only to callbacks-and-promises?